src/Servers/Kestrel/Core/src/Internal/Http/MessageBody.cs

```diff
@@ -154,13 +154,15 @@ protected void AddAndCheckConsumedBytes(long consumedBytes)
 
         protected ValueTask<ReadResult> StartTimingReadAsync(ValueTask<ReadResult> readAwaitable, CancellationToken cancellationToken)
         {
-
-            if (!readAwaitable.IsCompleted && _timingEnabled)
+            if (!readAwaitable.IsCompleted)
```

Comment:
@halter73 do these changes already exist in main?

Reply:
Yes. #31543

Interesting that it still shows up in the diff when the PR is targeting main.